frontend/amundsen_application/static/js/pages/HomePage/index.spec.tsx

```diff
@@ -4,11 +4,11 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 
-import Breadcrumb from 'components/Breadcrumb';
-import MyBookmarks from 'components/Bookmark/MyBookmarks';
-import PopularTables from 'components/PopularResources';
-import SearchBar from 'components/SearchBar';
-import TagsListContainer from 'components/Tags';
+import Breadcrumb from 'features/Widgets/Breadcrumb';
+import MyBookmarks from 'features/Widgets/MyBookmarks';
+import PopularTables from 'features/Widgets/PopularResources';
+import SearchBar from 'features/Widgets/SearchBar';
+import TagsListContainer from 'features/Widgets/Tags';
```

Comment:
Maybe there was a misunderstanding, but I didn't mean to move all components into a Widgets folder. 
We want to have only the Homepage widgets within a feature/HomepageWidgets folder.

Reply:
I rearranged the directory structure as we discussed, I think it's now how we want it?